docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -149,15 +149,25 @@ Each property described in the following paragraphs regarding the microservices
 }
 ```
 
-The serviceId **must** be in `kebab-case` format.
+The serviceId **must** be in `kebab-case` format and should not exceed the length of 63 characters.
 
 Here below are listed all the properties that you can provide for each microservice item:
   
 - **`itemId`**: a unique item id that can be used to identify the item and all the services generated from it. Each service created using this item will have the identifier value in the **sourceComponentId** property.
-- **`defaultEnvironmentVariables`**: the environment variables that will overwrite the default environment variables applied by DevOps Console.  
-  In particular, for each of them you need to provide:  
+- **`defaultEnvironmentVariables`**: the environment variables that will overwrite the default environment variables applied by the Console.  
+  In particular, for each of them you need to provide:
   - **`name`**: the variable name (generally, a key written in `UPPER_SNAKE_CASE`)
   - **`value`**: the variable default value; it can contain placeholders that will be replaced with the actual values when the service is created, as explained in the [interpolation section](#interpolating-default-labels-annotations-and-environment-variables-values).
+  - **`description`**: a brief description of the variable
+  - **`readOnly`**: a boolean that represents if you can change the value of the variable through the Console
+  You can also define a variable whose value will be provided by a kubernetes secret. In this case, you need to provide:
+  - **`valueType`**: must be set to `secret`
+  - **`secretName`**: the name of the secret that contains the value of the variable
+  - **`secretKey`**: the key of the secret that contains the value of the variable
+  - **`description`**: a brief description of the variable
+  - **`readOnly`**: a boolean that represents if you can change the value of the variable through the Console
+  Be aware that the secret must be created in the same namespace of the service will be deployed. Since the secret cannot be created through the Console, the user must be made aware of this requirement when they want to deploy the service.
```

Comment:
```suggestion
  Be aware that the secret must be created in the same namespace where the service will be deployed. Since the secret cannot be created through the Console, the user must be made aware of this requirement when they want to deploy the service.
```